Wrapping/Generators/Python/Tests/findEmptyClasses.py

```diff
@@ -70,22 +63,32 @@ def isEmpty(o):
            "HammingWindowFunction",
            "LanczosWindowFunction",
            "WelchWindowFunction",
+           "Tile"  # include from itkTileMontage remote module but only templated over dimension.
```

Comment:
Line 68 is the new item that needs to be ignored, but only when the remote module is present.

The other changes in this file are cosmetic and should have been put in a different patch set in an ideal development situation.